zlp-scheduler/app/views/admin/add_admin.html.haml

```diff
@@ -13,9 +13,9 @@
                 %td#specialTableData
                     = f.text_field :lastname
                 %td#specialTableData
-                    = f.text_field :email
+                    = f.email_field :email
```

Comment:
If I remember correctly, this should be a TAMU email, right?

Reply:
Yes.
I have updated it in 96c1d42cfd2769dd70032295fc00527f37cd9d7a, thanks!